assets/stylesheets/responsive/custom.scss

```diff
@@ -171,57 +171,89 @@ input.use-datepicker[type=text] {
   background:image-url('calendar.png') no-repeat 100px 5px $form_input-bg;
 }
 
-form input[type=submit], a.link_button_green, a.link_button_green_large {
-  text-decoration: none;
-  color: $button-color;
-  background-color: darken($button-bg, 5%);
+/* Button styles */
+
+@mixin button-base($size: normal, $disabled: false) {
+  padding: 0.5em 1.25em;
+  border-radius: 3px;
   display: inline-block;
-  cursor: pointer;
-  /* This style is expressed for all borders to avoid triggering an IE9 border rendering bug */
-  border-style: solid; border-color: darken($button-bg, 15%); border-width: 0 0 4px 0;
-  border-radius: 0;
-  padding: 3px 7px;
-  line-height: normal;
-  font-size: 1.1rem;
-  /* Same height as form elements */
-  height: 2.3125rem;
-  @include ie8 {
-     height: 2.3125em;
-     font-size: 1.1em;
-   }
-  vertical-align: top;
-  &:focus, &:hover{
+  font-weight: 600;
+  text-decoration: none;
+  transition: background-color 300ms ease-out;
+  margin-bottom: 0;
+  font-family: $font-family;
+  @if $size == large {
+    font-size: 2.2em;
+  }
+  &:hover,
+  &:active,
+  &:focus {
     text-decoration: none;
-    background-color: darken($button-bg, 10%);
-    border-bottom: 4px solid darken($button-bg, 20%);
+    transition: background-color 300ms ease-out;
+  }
+  @if $disabled {
+    opacity: 0.333;
+    &:hover,
+    &:active,
+    &:focus {
+      background-color: inherit;
+      color: inherit;
+      cursor: default;
+    }
   }
 }
 
-@include lte-ie7{
-
-  a.link_button_green, a.link_button_green_large {
-    padding: 0 7px;
-    height: 31px;
-    font-weight: bold;
+@mixin button-primary($args...) {
+  @include button-base($args...);
+  color: #fff;
+  background-color: $button-bg;
+  border: none;
+  &:hover,
+  &:active,
+  &:focus,
+  &:visited:hover,
+  &:visited:active,
+  &:visited:focus {
+      background-color: darken($button-bg, 10%);
+      color: #fff;
   }
+}
 
-  form input[type=submit] {
-    padding: 0;
-    height: 35px;
+@mixin button-secondary($args...) {
+  @include button-base($args...);
+    background-color: desaturate(darken($color_sand, 10%), 5%);
+    color: $color_black;
+    font-weight: normal;
+    &:hover,
+    &:active,
+    &:focus {
+      color: #333;
+      background-color:  desaturate(darken($color_sand, 15%), 5%);
+      color: $color_black;
+    }
+}
+
+@mixin button-tertiary($args...) {
```

Comment:
A tertiary button is almost invisible:

![screen shot 2015-08-07 at 14 13 44](https://cloud.githubusercontent.com/assets/282788/9136443/89da9ad8-3d0e-11e5-8ed3-5f30fce8c88a.png)

Disabled teritary:

![screen shot 2015-08-07 at 14 12 29](https://cloud.githubusercontent.com/assets/282788/9136424/68aeccb2-3d0e-11e5-8292-f3c2fddc42a8.png)